app/assets/javascripts/directives/search_sidebar.js

```diff
@@ -0,0 +1,170 @@
+angular.module("OpenHq").directive("searchSidebar", function(Search) {
+  return {
+    restrict: "E",
+    template: JST['templates/directives/search_sidebar'],
+
+    controller: function($scope) {
+      $scope.searching = false; // currently performing a seach
+      $scope.term = ""; // what the user has searched for
+      $scope.count = 0; // number of results returned
+      $scope.morePages = false; // the search results include more pages
+      $scope.page = 0; // current page of results
```

Comment:
A lot of these are ok without a comment and changing this one to `$scope.currentPage` removes the need for comment